bellows/types/named.py

```diff
@@ -209,8 +229,16 @@ class EzspConfigId(basic.uint8_t, enum.Enum):
     CONFIG_ZLL_GROUP_ADDRESSES = 0x2F
     # ZLL rssi threshold initial configuration.
     CONFIG_ZLL_RSSI_THRESHOLD = 0x30
+    # ??? - TODO.
```

Comment:
Docs have: "The maximum number of pairings
supported by the stack. Controllers
must support at least one pairing
table entry while targets must
support at least five. "